docs/Deploy/overview.mdx

```diff
@@ -1,17 +1,29 @@
 ---
-title: "Deploying Conduit"
-sidebar_label: "Overview"
+title: "Deploying Conduit on your machine"
+sidebar_label: "Local Deployment"
 sidebar_position: 1
 ---
 
-# Deploying Conduit
+# Deploying Conduit on your Machine
```

Comment:
Do we want to mention nightly builds here?

Reply:
mentioned it in the binary downloads